index.js

```diff
@@ -0,0 +1,6 @@
+
+
+function show_alert()
+{  
```

Comment:
When writing functions you want to be careful of your indentation. The function itself is fine, but syntax wise you should write it more like 
`
function showAlert() {
  alert("Welcome to all my new skils")
}
`